### PR TITLE
Flaggy <Flag/>

### PR DESCRIPTION
**Pull request by thesephist**
wat if orpheus wave dat flag when u hover over the logo

flaggy-flag

### CODE REVIEW

**Review by lachlanjc — CHANGES_REQUESTED** [review]
Looks awesome. Two small accessibility changes needed: reset the animation on theme.mediaQueries.reduceMotion, then add :focus styling.

File: src/components/Flag.js [hunk 1]
```diff
@@ -14,6 +32,13 @@ const Flag = styled(Link)`
     width: 144px;
     height: 72px;
   }
+  &:hover,
+  &:focus {
+    animation: ${waveFlag} 0.5s linear infinite alternate;
+  }
+  ${theme.mediaQueries.reduceMotion} {
+    animation: none;
```
Comment (lachlanjc) [inline src/components/Flag.js]:
```suggestion
    animation: none !important;
```

File: src/components/Flag.js [hunk 2]
```diff
@@ -22,6 +47,13 @@ const Flag = styled(Link)`
       ${theme.mediaQueries.md} {
         height: 54px !important;
       }
+      &:hover,
+      &:focus {
+        animation: ${waveFlagScaled} 0.5s linear infinite alternate;
+      }
+      ${theme.mediaQueries.reduceMotion} {
```
Comment (lachlanjc) [inline src/components/Flag.js]: I think you can remove this whole selector

Comment (thesephist) [reply src/components/Flag.js]: Touche.

**Comment by lachlanjc** [comment]
On a touchscreen on the homepage, if you tap the flag, it starts waving & never stops. :)

This is funny the first time but not being able to disable it without leaving the page is…not ideal.

**Review by lachlanjc — APPROVED** [review]

**Comment by zanedb** [comment]
@thesephist you and your flaggy-flag promoting ways. you are truly a rebellious Tim.
(sorry for interrupting actual work being done here)

**Comment by thesephist** [comment]
@zanedb haha i was hoping a fellow Tim would stumble upon this timfoolery. Worth it. 🚀

**Pull request merged**